packages/framework/src/Model/Product/Search/FilterQueryFactory.php

```diff
@@ -238,10 +238,15 @@ public function createVisibleProductsByProductIdsFilter(array $productIds): Filt
 
     /**
      * @param int[] $productIds
+     * @param int|null $limit
      * @return \Shopsys\FrameworkBundle\Model\Product\Search\FilterQuery
      */
-    public function createSellableProductsByProductIdsFilter(array $productIds): FilterQuery
+    public function createSellableProductsByProductIdsFilter(array $productIds, ?int $limit = null): FilterQuery
     {
-        return $this->createVisibleProductsByProductIdsFilter($productIds)->filterOnlySellable();
+        $filterQuery = $this->createVisibleProductsByProductIdsFilter($productIds)->filterOnlySellable();
+        if ($limit === null) {
+            return $filterQuery;
+        }
```

Comment:
Please separate this part of code from others by new lines

Reply:
I do not see any need to add 2 more blank lines. This is pretty straightforwaqrd as is.

Reply:
Ok, I have added it 😉